Modules/Core/ImageFunction/include/itkBSplineInterpolateImageFunction.h

```diff
@@ -312,6 +315,13 @@ class ITK_TEMPLATE_EXPORT BSplineInterpolateImageFunction:
   itkGetConstMacro(UseImageDirection, bool);
   itkBooleanMacro(UseImageDirection);
 
+  SizeType GetRadius() const override
+    {
+    SizeType radius;
+    radius.Fill( m_SplineOrder + 1 );
+    return radius;
```

Comment:
For what it's worth, this function could be written as `return SizeType::Filled(m_SplineOrder + 1);`